astropy/samp/tests/test_standard_profile.py

```diff
@@ -13,6 +14,7 @@ def setup_module(module):
     conf.use_internet = False
 
 
+@pytest.mark.skipif(CI, reason="flaky in CI")
```

Comment:
have you thought about using pytest rerunfailures instead? I had a reasonably good experience with it in astroquery/pyvo for flaky tests that would usually pass after a CI rerun.

Reply:
Not sure if re-running this adds any benefit as a whole. SAMP code is pretty stable by now and slated to be moved over to pyvo at some point. But I'll keep that in mind for other stuff in the future. Thanks!